requests_auth/authentication.py

```diff
@@ -98,8 +98,9 @@ def __and__(self, other):
 class BrowserAuth:
     def __init__(self, kwargs):
         """
+        :param redirect_uri_domain: FQDN for localhost. localhost by default.
```

Comment:
the documentation should be the same on all classes. It should also be reflected in README